news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
```

Comment:
restore to tumblr link except use 163075746433 as the id

see previous news-articles for example